web/src/app/orbs/acidic/page.tsx

```diff
@@ -0,0 +1,25 @@
+import OrbLayout from "../components/orb-layout/orb-layout";
+
+export default function AcidicPage() {
+  return (
+    <OrbLayout
+      title="Acidic"
+      description="Activating this will dissolve all enemy pieces in your path, and make the tile unusable. Available in three variants: row, column, radial."
```

Comment:
In all powerups I'd skip the `Available in three variants` and the below list of what each variant does, since you already added multiDimensional and IMHO that should be more than enough